cli/api/service.go

```diff
@@ -167,6 +167,34 @@ func (a *api) GetServiceStatus(serviceID string) (map[string]map[string]interfac
 				row["Cur/Max/Avg"] = fmt.Sprintf("--")
 
 				rowmap[fmt.Sprintf("%s/%d", svc.ID, stat.State.InstanceID)] = row
+
+				if stat.Status == dao.Running && len(stat.HealthCheckStatuses) > 0 {
+
+					explicitFailure := false
+
+					for hcName, hcResult := range stat.HealthCheckStatuses {
+						newrow := make(map[string]interface{})
```

Comment:
That is how the other rows were defined.  See lines 128 and 148.  Also, the API function is expected to return a map[string]map[string]interface{}